setup.py

```diff
@@ -5,18 +5,31 @@
 """
 import sys
 
+# try:
+#   import numpy
+# except ImportError:
+#   print("Numpy is needed for running and building of PHOEBE")
+#   sys.exit(1)
+
 try:
   import numpy
+  import setuptools
+  import packaging
 except ImportError:
-  print("Numpy is needed for running and building of PHOEBE")
+  print("Numpy, setuptools and packaging are needed for the running and building of PHOEBE")
```

Comment:
setuptools and packaging should both come with python, right?  I just wonder if we need to actually determine which the user doesn't have and just list those to avoid confusion?